README.md

```diff
@@ -0,0 +1,145 @@
+# ReactBrightcovePlayer
+A React component to load a brightcove player in the browser.
+
+## Table of Contents
+
+<!-- START doctoc -->
+<!-- END doctoc -->
+## Installation
+
+```sh
+npm install --save react-brightcove-player
+```
+
+## Usage
+
+To include `react-brightcove-player` on your website or web application, use any of the following methods.
+
+### Via `<script>` Tags
+
+Get the script however you prefer. The `ReactBrightcovePlayer` component will be exported on the global `window` object. From there you can render it with `react` and `react-dom`, but those are not required to include the component on the page.
+
+> NOTE: Again React/ReactDOM are **NOT** required, they are only used to show a working example.
+
+```html
+<div id='fixture'></div>
+<script src="//path/to/react.min.js"></script>
+<script src="//path/to/react-dom.min.js"></script>
+<script src="//path/to/react-brightcove-player.min.js"></script>
+<script>
+
+  var React = window.React;
+  var ReactDOM = window.ReactDOM;
+  var ReactBrightcovePlayer = window.ReactBrightcovePlayer;
+
+  var reactBrightcovePlayer = window.reactBrightcovePlayer = ReactDOM.render(
+    React.createElement(ReactBrightcovePlayer, {
+      accountId: '1234678',
+      onSuccess: function(success) {
+        // two ways to get the underlying player/iframe at this point.
+        console.log(success.ref)
+        console.log(reactBrightcovePlayer.state.player);
+      }
+    }),
+    document.getElementById('fixture')
+  );
+
+</script>
+```
+
+### CommonJS
+
+When using with Browserify/rollup/webpack, install via npm and `require` the plugin as you would any other module.
+
+> NOTE: React/ReactDOM/global are **NOT** required, they are only used to show a working example.
+
+```js
+var React = require('react');
+var ReactDOM = require('react-dom');
+var document = require('global/document');
+var ReactBrightcovePlayer = require('react-brightcove-player');
+
+
+var reactBrightcovePlayer = ReactDOM.render(
+  React.createElement(ReactBrightcovePlayer, {
+    accountId: '1234678',
+    onSuccess: function(success) {
+      // two ways to get the underlying player/iframe at this point.
+      console.log(success.ref)
+      console.log(reactBrightcovePlayer.state.player);
+    }
+  }),
+  document.getElementById('fixture')
+);
+
+```
+
+### ES Module
+
+When using with Browserify/rollup/webpack, install via npm and `import` the plugin as you would any other module.
```

Comment:
It's best to avoid language like `as you would any other module` because not everyone necessarily has installed other modules. Best to be simple and explicit about things. If steps are equivalent for multiple sections can be pulled out into another one or linked to another tutorial.